client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -43,7 +43,10 @@ const appShape = PropTypes.shape({
 
```

Comment:
__Explanation__
I have added `id="appDetail-wrapper"` to several elements that should use its styles for consistent behavior. To allows differences in the style, I have added classes like `has-message` and `has-external-app`.